app/forms/declaration/Choice.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+import play.api.data.Forms.text
+import play.api.data.{Form, Forms}
+import play.api.libs.json.Json
+import utils.validators.forms.FieldValidator.{isContainedIn, isEmpty, nonEmpty, _}
+
+case class Choice(addItem: String)
```

Comment:
This name should be changed. We have Choice in `app/forms/`, another one in `app/forms/declarations` will be difficult to maintain, from the beggining of the PR I had a problem with  thinking about Choice in different way like Choice from the first screen. It can be SealChoice ContainerChoice, something like this, I don't know what this choice mean :)

Reply:
Good point about the choice of name!   I deliberately made it vague because it is simply the answer to a "yes/no" question and could in theory be used anywhere.   I originally called it an "AddItem" (as in, "do you want to add an item") as can be seen in the member variable name.

How about `YesNoAnswer` for the name of this form mapping object?

Reply:
I think it's okay. It can be reused in more places if necessary. We can ask team for their thoughts, but for me it's okay.

Reply:
Hmm... Maybe having two case object with custom unapply would be Good enough.